api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -117,9 +117,12 @@ message Bootstrap {
   // Optional watchdog configuration.
   Watchdog watchdog = 8;
 
-  // Configuration for an external tracing provider. If not specified, no
-  // tracing will be performed.
-  trace.v2.Tracing tracing = 9;
+  // Configuration for an external tracing provider.
+  //
+  // .. attention::
+  //  This field has been deprecated in favor of :ref:`HttpConnectionManager.Tracing.provider
+  //  <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.Tracing.provider>`.
+  trace.v2.Tracing tracing = 9 [deprecated = true];
```

Comment:
v2 is frozen so this need to move to v3. Please merge master.

Reply:
Can I at least update the doc ?

Otherwise, the doc will be confusing. Currently, it says: 

```
// If not specified, no tracing will be performed.	
```

It's not possible to understand from it that a configuration inside HCM is enough.

Reply:
Yes the doc fix is fine, thanks.

Reply:
fixed